configure.ac

```diff
@@ -324,6 +324,16 @@
         CFLAGS="$CFLAGS -std=gnu99"
     fi
 
+    # check if our compiler supports -Wunused-macros
+    AC_MSG_CHECKING(for -Wunused-macros support)
```

Comment:
What do you think about this ?

Should we do it for other warning flags ?

Reply:
I don't think these belong in configure, other than, perhaps, as a `--enable-strict-compiler-flags` or similar. But generally we're not supposed to do too much with the CFLAGS. We probably already do too much right now.

Reply:
Ok for something like `--enable-rust-strict`

But then, how do I get CI to get the most of it, as is done here ?